route/config/route_config_params.yaml

```diff
@@ -4,4 +4,10 @@ max_crosstrack_error: 25
 
 # Double: downtrack_range
 # Units: meter
-destination_downtrack_range: 0
\ No newline at end of file
+destination_downtrack_range: 0
+
+# Int: cross-track deviation counter
```

Comment:
Is this measured in timesteps?

Reply:
Also would be good for both of these to describe what happens when the count reaches these thresholds, so that someone modifying these would know what is changing.

Reply:
I included the explanation in the route_generator_worker.cpp file where they are implemented, but I'll included here as well.